packages/cactus-plugin-keychain-vault/src/main/typescript/prometheus-exporter/metrics.ts

```diff
@@ -0,0 +1,10 @@
+import { Gauge } from "prom-client";
+
+export const K_CACTUS_KEYCHAIN_VAULT_TOTAL_KEY_COUNT =
+  "cactus_keychain_vault_total_key_count";
+
+export const totalKeyCount = new Gauge({
```

Comment:
@jagpreetsinghsasan We might need a different name/description for this that properly conveys the fact that the vault itself could have millions of keys in it that are pre-existing and were not set by the Cactus keychain plugin (and therefore the counter doesn't know about their existence either).
Example: Big enterprise deploys Cactus and they connect it to the existing Vault deployment that they had been running for years and dozens of other (unrelated) applications of theirs store thousands of secrets in that Vault instance. In this case if someone looked at our metrics and saw that the "total keys present in vault" metrics is, let's say `5` they could become very, very concerned that either Cactus wiped out the rest of the keys or something else has gone very wrong (if our metrics to be taken at face value).

I'm thinking maybe something like this could prevent the people in the above example from having a stressful time:
`The number of keys that were set in the backing Vault deployment via this specific keychain plugin instance.`

For the metric name, not sure, it's tough to name things... maybe something like this (but feel free to suggest something else)
`K_CACTUS_KEYCHAIN_VAULT_MANAGED_KEY_COUNT`
I replaced the `TOTAL` with `MANAGED` trying to convey the information that this is the key count that's under the management of this plugin, or something like that. Just an idea, open to going with something else.

Reply:
This makes perfect sense. The enterprise can decide to use existing vault for cactus purpose and this name will then be troublesome. I shall fix it